source/whats-new.txt

```diff
@@ -37,6 +38,15 @@ Upcoming Breaking Changes
   aggregation framework and removed in a future release.
 - Beginning with v5.0, the {+driver-short+} will require Java 11 or later.
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+New features of the 4.11 driver release include:
+
+- Upgraded version of the optional Netty dependency to 4.1.87.Final 
```

Comment:
Suggestion:
Since it's a driver build dependency for java-core (https://github.com/mongodb/mongo-java-driver/blob/master/build.gradle#L53), I think it would be good to focus the explanation on how it might impact the user. Currently, the only impact is to TLS/SSL when using Netty SslContext.

E.g.
If you use the Netty plugin for SslContext, note that the driver now tests with v4.187.Final of the io.netty:netty-all package.